client/src/app/site/pages/meetings/modules/projector/modules/slides/components/motions/modules/motion-block-slide/components/motion-block-slide/motion-block-slide.component.ts

```diff
@@ -91,13 +91,16 @@ export class MotionBlockSlideComponent extends BaseMotionSlideComponent<MotionBl
         return this.makeIndicesArray(this.columns);
     }
 
-    public constructor(private meetingSettingsService: MeetingSettingsService) {
+    public constructor(private meetingSettingsService: MeetingSettingsService, private cd: ChangeDetectorRef) {
         super();
         this.languageCollator = getIntlCollatorForLang(this.translate.currentLang);
 
         this.meetingSettingsService
             .get(`motions_block_slide_columns`)
-            .subscribe(value => (this.maxColumns = value > 0 ? value : MAX_COLUMNS));
```

Comment:
Normally I would not approve this, as the observable is not unsubscribed. It would be better to provide an observable to the template by modifying the value via `map` to fix this problem.
Anyway, as the projector gets replaced anyway I don't want to bother with this.